generate_figures/deep_sets_linear_profiling_var_n.py

```diff
@@ -2,103 +2,51 @@
-        else:
-            d.append(get) 
-    return(pd.DataFrame.from_records(d))
-
-df_deep = satisfying_runs("baseline_deep_sets_N")            
-
-##once we have retcode will be something like 
-#df_successes= df_deep.loc[df_deep['ret_code']==0]
-
-
-df_successes= df_deep.loc[df_deep['retcode']==0]
-
-#making the dataframe for training time quartiles 
-quantiles= [0.1,0.25,0.5,0.75,0.9]
-
-
-
-quant_train_time_deep = df_successes.groupby('N').quantile(quantiles)['train_time'].unstack(level=-1)
-quant_train_time_deep.reset_index(inplace=True)
-quant_train_time_deep.columns = ['N'] + [f'quantile_{q}' for q in quantiles]
-
-
-#making the dataframe for rel_error quartiles
-# 
-quant_test_u_rel_error_deep = df_successes.groupby('N').quantile(quantiles)['test_u_rel_error'].unstack(level=-1)
-quant_test_u_rel_error_deep.reset_index(inplace=True)
-quant_test_u_rel_error_deep.columns = ['N'] + [f'quantile_{q}' for q in quantiles]
-
-plt.rcParams.update(params) 
-
-
-ax_time = plt.subplot(121)
-plt.plot(quant_train_time_deep['N'], quant_train_time_deep['quantile_0.5'], label= r"Median")
-plt.fill_between(quant_train_time_deep['N'], quant_train_time_deep["quantile_0.1"],quant_train_time_deep["quantile_0.9"], color ='cornflowerblue', alpha=0.2, label = r"$10$th and $90$th percentiles")
-plt.fill_between(quant_train_time_deep['N'], quant_train_time_deep["quantile_0.25"],quant_train_time_deep["quantile_0.75"], color ='cornflowerblue', alpha=0.6, label = r"$25$th and $75$th percentiles")
-ax_time.set_xscale('log')
-ax_time.xaxis.set_ticks([50, 100, 1000, 10000, 100000])
-plt.title(r"Computation time (seconds)")
-plt.xlabel(r"N")
-plt.legend(prop={"size": fontsize}, loc='upper left')
-plt.tight_layout()
-
-
-
-ax_loss = plt.subplot(122)
-plt.plot(quant_test_u_rel_error_deep['N'], quant_test_u_rel_error_deep['quantile_0.5'], label=r"Median")
-plt.fill_between(quant_test_u_rel_error_deep['N'], quant_test_u_rel_error_deep["quantile_0.1"],quant_test_u_rel_error_deep["quantile_0.9"], color ='cornflowerblue', alpha=0.2, label = r"$10$th and $90$th percentiles")
-plt.fill_between(quant_test_u_rel_error_deep['N'], quant_test_u_rel_error_deep["quantile_0.25"],quant_test_u_rel_error_deep["quantile_0.75"], color ='cornflowerblue', alpha=0.6, label = r"$25$th and $75$th percentiles")
-ax_loss.set_xscale('log')
-ax_loss.set_yscale('log')
-ax_loss.xaxis.set_ticks([50, 100, 1000, 10000, 100000])
-ax_loss.yaxis.set_ticks([0.0001, 0.001]) 
-plt.title(r"Policy errors ($\epsilon_{\mathrm{rel}}$)")
-plt.xlabel(r"N")
-plt.legend(prop={"size": fontsize}, loc='lower left')
-plt.tight_layout()
-
-plt.savefig(output_path)
+project = "highdimensionaleconlab/symmetry_dynamic_programming"         
+
+
+df_deep = get_results_by_tag(api, project, "baseline_deep_sets_N", get_config=True)
+assert(df_deep.id.nunique() == 1000)
+
+df_deep = df_deep[df_deep["retcode"] == 0]
+plot_dict = {'train_time':{'ticks_x':[50, 100, 1000, 10000, 100000], 'ticks_y':[20, 100],
```

Comment:
See here is the weird dictionary. I can revert back because its a bit confusing and it was literally only being repeated twice....it was more just to try and see if I could do it.